sdk/python/feast/registry.py

```diff
@@ -143,44 +213,60 @@ def clone(self) -> "Registry":
         new_registry._registry_store = NoopRegistryStore()
         return new_registry
 
+    def to_repo_contents(self, project: str) -> RepoContents:
```

Comment:
Seems like a confusing concept to me. What is in the registry during `plan` _should_ be different from the repo itself, but we map both registry contents + the current repo python objects to repocontents?

not sure what naming is better for RepoContents... esp since we want to moev away from the term Fco

Reply:
should be fixed since we are now comparing the current registry with the new repo contents